### ISSUE TITLE
[BUG] server update not working

### ISSUE THREAD

**Issue by Bugstir** [phpservermon/phpservermon#1082]
Having problem with server status being updated. Cron job set for 2-minute intervals. During testing of renaming a website folder to trigger a status change, you would be alerted when the Cron job has completed and verified the change. 
But after the folder has been renamed correctly it takes 60 minutes for the website GUI to show the correct status.
However if at any time I click the update button manually it'll show the correct status.

So basically the behind-the-scenes update feature is not working correctly if the cron job is doing what it's supposed to do behind the scenes no alerts are triggered because the update feature is not grabbing the current information at hand.

It seems that either there's something not correctly coded or we should create an additional PHP file to execute the update???

**Comment by wienfuchs** [comment]
Cannot confirm this behaviour. Also you missed relevant information like versions you use (php, phpservermon for example, cron call command) and so on. So: Server update in fact _is_ working. Maybe _your environment_ breaks this behaviour.

**Comment by Bugstir** [comment]
Yeah sorry for the lack of info seeing how I thought there's only one version 3.2.0 that actually works or somewhat stable. System technically on a centOS had no problems with installation no errors database working cron job working. not to mention there's a lot of internet chatter about the same thing but I figured I would come straight to the source of the file.... Php5.6.
